### DISCUSSION TITLE
Different senderSignature

### DISCUSSION THREAD

**Discussion by opaque01** [derhansen/sf_event_mgt#1031] [Q&A] [answered]
Hallo,

is there a way to add different sender signatures depending on the event category?

Best regards
opaque

**Comment by derhansen** [comment, marked as answer]
No, not by default. You can try to add a condition for the signature in the email template like `<f:if condition="{event.category.0.uid} == 1">...your signature for category UID 1</f:if>`

**Reply by opaque01** [reply]
Hallo

thx, works perfect.

Best regards
opaque
